homeworks/taras.bobov_LotusElize/GamesProject/pages/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
That is what I see, when open your page 
The typical files structure for static html site is this

- folder for styles
- folder for images
- index.html
- any_other_files.html

<img width="1427" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/a8329dfc-3048-47b0-95f1-4b8b861ca80d">

Reply:
Please correct the file structure or update path to styles and relative page, I can't check your work now.
The index.html should contain about info, so no need to create for it separate html page

Reply:
@vladyslav-yermolin-moc Thanks, done refactored , in new commit for this pull request  [2-css_LotusElize (refactored)](https://github.com/MastersAcademy/fe-course-basic-2023/pull/143/commits/592a5154278040c7f20b74900969a781bdcf5576)